README.md

```diff
@@ -163,39 +163,45 @@ cargo pgx init
-`timestamp with time zone` | `pgx::TimestampWithTimeZone`
-`anyarray` | `pgx::AnyArray`
-`anyelement` | `pgx::AnyElement`
-`box` | `pgx::pg_sys::BOX`
-`point` | `pgx::pgx_sys::Point`
-`tid` | `pgx::pg_sys::ItemPointerData`
-`cstring` | `&core::ffi::CStr`
-`inet` | `pgx::Inet(String)` -- TODO: needs better support
-`numeric` | `pgx::Numeric<P, S> or pgx::AnyNumeric`
-`void` | `()`
-`ARRAY[]::<type>` | `Vec<Option<T>>` or `pgx::Array<T>` (zero-copy)
-`NULL` | `Option::None`
-`internal` | `pgx::PgBox<T>` where `T` is any Rust/Postgres struct
-`uuid` | `pgx::Uuid([u8; 16])`
+| Postgres Type              | Rust Type (as `Option<T>`)                            |
+|----------------------------|-------------------------------------------------------|
+| `bytea`                    | `Vec<u8>` or `&[u8]` (zero-copy)                      |
+| `text`                     | `String` or `&str` (zero-copy)                        |
+| `varchar`                  | `String` or `&str` (zero-copy) or `char`              |
+| `"char"`                   | `i8`                                                  |
+| `smallint`                 | `i16`                                                 |
+| `integer`                  | `i32`                                                 |
+| `bigint`                   | `i64`                                                 |
+| `oid`                      | `u32`                                                 |
+| `real`                     | `f32`                                                 |
+| `double precision`         | `f64`                                                 |
+| `bool`                     | `bool`                                                |
+| `json`                     | `pgx::Json(serde_json::Value)`                        |
+| `jsonb`                    | `pgx::JsonB(serde_json::Value)`                       |
+| `date`                     | `pgx::Date`                                           |
+| `time`                     | `pgx::Time`                                           |
+| `timestamp`                | `pgx::Timestamp`                                      |
+| `time with time zone`      | `pgx::TimeWithTimeZone`                               |
+| `timestamp with time zone` | `pgx::TimestampWithTimeZone`                          |
+| `anyarray`                 | `pgx::AnyArray`                                       |
+| `anyelement`               | `pgx::AnyElement`                                     |
+| `box`                      | `pgx::pg_sys::BOX`                                    |
+| `point`                    | `pgx::pgx_sys::Point`                                 |
+| `tid`                      | `pgx::pg_sys::ItemPointerData`                        |
+| `cstring`                  | `&core::ffi::CStr`                                    |
+| `inet`                     | `pgx::Inet(String)` -- TODO: needs better support     |
```

Comment:
See previous remark.

Reply:
yes.  All these were there before you're just noticing them b/c intellij forced me to reformat the table.